tutorials/build-smart-contract-thirdweb/TUTORIAL.md

```diff
@@ -0,0 +1,97 @@
+
+   ```bash
+   // SPDX-License-Identifier: MIT
+   pragma solidity ^0.8.0;
+
+   import "@thirdweb-dev/contracts/base/ERC721Base.sol";
+
+   contract Contract is ERC721Base {
+       constructor(
+           string memory _name,
+           string memory _symbol,
+           address _royaltyRecipient,
+           uint128 _royaltyBps
+       ) ERC721Base(_name, _symbol, _royaltyRecipient, _royaltyBps) {}
+   }
+   ```
+
+   This contract inherits the functionality of ERC721Base through the following steps:
+
+   - Importing the ERC721Base contract
+   - Inheriting the contract by declaring that our contract is an ERC721Base contract
+   - Implementing any required methods, such as the constructor.
+
+5. After modifying your contract with your desired custom logic, you may deploy it to zkSync Era using [Deploy](https://portal.thirdweb.com/deploy?utm_source=zksync&utm_medium=docs&utm_campaign=chain_docs).
+
+---
+
+Alternatively, you can deploy a prebuilt contract for NFTs, tokens, or marketplace directly from the thirdweb Explore page:
+
+1. Go to the thirdweb Explore page: [https://thirdweb.com/explore](https://thirdweb.com/explore?utm_source=zksync&utm_medium=docs&utm_campaign=chain_docs)
+
+   ![thirdweb Explore](./images/thirdweb-explore.jpg)
+
+2. Choose the type of contract you want to deploy from the available options: NFTs, tokens, marketplace, and more.
+3. Follow the on-screen prompts to configure and deploy your contract.
+
+> For more information on different contracts available on Explore, check out [thirdweb’s documentation.](https://portal.thirdweb.com/pre-built-contracts?utm_source=zksync&utm_medium=docs&utm_campaign=chain_docs)
+
+### Step 2 — Deploy Contract
+
+Deploy allows you to deploy a smart contract to any EVM compatible network without configuring RPC URLs, exposing your private keys, writing scripts, and other additional setup such as verifying your contract.
+
+1. To deploy your smart contract using deploy, navigate to the root directory of your project and execute the following command:
+
+   ```bash
+   npx thirdweb deploy
+   ```
+
+   Executing this command will trigger the following actions:
+
+   - Compiling all the contracts in the current directory.
+   - Providing the option to select which contract(s) you wish to deploy.
+   - Uploading your contract source code (ABI) to IPFS.
+
+2. When it is completed, it will open a dashboard interface to finish filling out the parameters.
+   - `_name`: contract name
+   - `_symbol`: symbol or "ticker"
+   - `_royaltyRecipient`: wallet address to receive royalties from secondary sales
+   - `_royaltyBps`: basis points (bps) that will be given to the royalty recipient for each secondary sale, e.g. 500 = 5%
+3. Select `ZksyncEra` as the network
+4. Manage additional settings on your contract’s dashboard as needed such as uploading NFTs, configuring permissions, and more.
```

Comment:
To me, after (creating an API key on the webUI and) running `npx thirdweb deploy` and filling the fields returned an error: 

```bash
Failed to deploy contract
Contract bytecode is invalid. 0x{"error":"Not Found"}
If you believe this is incorrect or the error persists, please reach out in [discord](https://discord.gg/thirdweb).
```

Reply:
@avneesh0612 can you confirm if `npx thidweb deploy` uses the zksolc compiler behind the scenes? All contracts deployed on zkSync Era must be compiled with our compilers